pkg/cfn/manager/nodegroup.go

```diff
@@ -136,15 +172,72 @@ func (c *StackCollection) ScaleNodeGroup(id int) error {
 	return c.UpdateStack(name, "scale-nodegroup", descriptionBuffer.String(), []byte(template), nil)
 }
 
-func (c *StackCollection) getStackTemplate(stackName string) (string, error) {
-	input := &cfn.GetTemplateInput{
-		StackName: aws.String(stackName),
+// GetNodeGroupSummaries returns a list of summaries for the nodegroups of a cluster
+func (c *StackCollection) GetNodeGroupSummaries() ([]*NodeGroupSummary, error) {
+	stacks, err := c.ListStacks(fmt.Sprintf("^(eksctl|EKS)-%s-nodegroup-.+$", c.spec.Metadata.Name), cfn.StackStatusCreateComplete)
```

Comment:
This also needs to include `cfn.StackStatusUpdateComplete` and possibly other status...

Reply:
Confirmed this is addressed in 04c0d6a. I've fixed the failing test cases due to that in 
5ad572a